### PR TITLE
Issue #SB-16868 fix: removed hardcode redirection to resource page on…

### PR DESCRIPTION
**Pull request by anoophm**
… stay in browser button click

### CODE REVIEW

File: src/app/client/src/app/modules/dial-code-search/components/dial-code/dial-code.component.ts [hunk 1]
```diff
@@ -433,17 +433,12 @@ export class DialCodeComponent implements OnInit, OnDestroy {
   }
   public handleCloseButton() {
     if (_.get(this.activatedRoute, 'snapshot.queryParams.textbook') && _.get(this.dialCodeService, 'dialCodeResult.count') > 1) {
-      this.router.navigate(['/get/dial', _.get(this.activatedRoute, 'snapshot.params.dialCode')]);
+      return this.router.navigate(['/get/dial', _.get(this.activatedRoute, 'snapshot.params.dialCode')]);
+    }
+    if (this.userService.loggedIn) {
+      this.navigationHelperService.navigateToPreviousUrl('/resources');
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/dial-code-search/components/dial-code/dial-code.component.ts:439]: Why did you removed the check for "play" in the URL? Check with @yravinderkumar33 on this implementation.

File: src/app/client/src/app/modules/public/module/player/components/public-collection-player/public-collection-player.component.ts [hunk 2]
```diff
@@ -338,16 +338,7 @@ export class PublicCollectionPlayerComponent implements OnInit, OnDestroy, After
...
-       const  previousUrl =  this.navigationHelperService.getPreviousUrl();
-       if (Boolean(_.includes(previousUrl.url, '/play/collection/'))) {
-        return this.router.navigate(['/']);
-       }
-       // tslint:disable-next-line: max-line-length
-       previousUrl.queryParams ? this.router.navigate([previousUrl.url], {queryParams: previousUrl.queryParams}) : this.router.navigate([previousUrl.url]);
-      } else {
-        this.navigationHelperService.navigateToResource('/explore');
-      }
+      this.navigationHelperService.navigateToPreviousUrl('/explore');
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/public/module/player/components/public-collection-player/public-collection-player.component.ts:341]: You have removed complete offline check here. Is it not require now?

File: src/app/client/src/app/modules/dial-code-search/components/dial-code/dial-code.component.ts (hunk 1 above)
Comment (anoophm) [reply src/app/client/src/app/modules/dial-code-search/components/dial-code/dial-code.component.ts:439]: its not needed. In the history /play url will not be there. Please check the logic of navigationHelperService.

File: src/app/client/src/app/modules/public/module/player/components/public-collection-player/public-collection-player.component.ts (hunk 2 above)
Comment (anoophm) [reply src/app/client/src/app/modules/public/module/player/components/public-collection-player/public-collection-player.component.ts:341]: I have moved offline code to [desktop-repo ](https://github.com/Sunbird-Ed/sunbird-desktop-app/tree/master/src/client)in the previous release. We need to remove all references of offline code in all the components.

**Pull request merged**